extensions/wikia/WikiFactory/api/MarkWikiAsClosedController.class.php

```diff
@@ -30,32 +30,48 @@ public function markWikiAsClosed() {
 		if ( !is_numeric( $wikiId ) || empty( $reason ) || !is_numeric( $userId ) ) {
 			// No wikiId, userId or reason given: Bad Request
 			$this->response->setCode( 400 );
-			$this->info('no wikiId or reason parameter in request');
+			$this->info( 'no wikiId or reason parameter in request' );
 
 			return;
 		}
 
-		if ( !empty( $fandomCreatorCommunityId ) && !is_numeric( $fandomCreatorCommunityId ) ) {
-			$this->response->setCode( 400 );
-			$this->info('invalid  fandomCreatorCommunityId parameter in request');
+		if ( !empty( $fandomCreatorCommunityId ) ) {
+			if ( !static::isValidFandomCreatorCommunityId( $fandomCreatorCommunityId, $wikiId )) {
+				$this->response->setCode( 400 );
+				$this->info( 'invalid fandomCreatorCommunityId parameter in request: ' . $fandomCreatorCommunityId );
```

Comment:
I think it is better to put variables into the log message context like `$this->info( 'message', [ 'fandomCreatorCommunityId' => $fandomCreatorCommunityId] );` 
It makes using Kibana logs easier: one can filter out some types of error messages or search by context field values. Same applies to other log messages in this PR.

Reply:
Done.